RELEASE-NOTES.md

```diff
@@ -1,5 +1,63 @@
 # Prefect Release Notes
 
+## Release 2.18.1
+
+### Fixes
+- Fix improper context access for nested async task outside of flow — https://github.com/PrefectHQ/prefect/pull/12810
+- Fix using default interval schedule in `prefect deploy` — https://github.com/PrefectHQ/prefect/pull/12833
+- Handle case in `validationUpdate` schema where definitions are falsy — https://github.com/PrefectHQ/prefect/pull/12880
+- Allow `prefect cloud login` to override current workspace — https://github.com/PrefectHQ/prefect/pull/12867
+- Remove extra quotes in `prefect deployment run --watch` — https://github.com/PrefectHQ/prefect/pull/12894
+
+### Experimental
+
+#### Events and Automations
+- Support filtering by automation name:
+    - https://github.com/PrefectHQ/prefect/pull/12850
+    - https://github.com/PrefectHQ/prefect/pull/12884
+    - https://github.com/PrefectHQ/prefect/pull/12887
+- Add support for using the "normal" Trigger classes for `flow.serve` and `.deploy` — https://github.com/PrefectHQ/prefect/pull/12789
+- Add an account-level event subscriber — https://github.com/PrefectHQ/prefect/pull/12808
+- Emit flow run state change events — https://github.com/PrefectHQ/prefect/pull/12825
+- Emit deployment status persistence and events — https://github.com/PrefectHQ/prefect/pull/12853
+- Enable event streaming from `PrefectCloudEventSubscriber` via CLI  — https://github.com/PrefectHQ/prefect/pull/12796
+- Update the `prefect automation delete` CLI — https://github.com/PrefectHQ/prefect/pull/12876
+
+#### Engine
+- Add new experimental engine for tasks and flows with improved readability and extensibility — https://github.com/PrefectHQ/prefect/pull/12856
+
+### Documentation
+- Improve installation instructions  — https://github.com/PrefectHQ/prefect/pull/12783
+- Improve quickstart  — https://github.com/PrefectHQ/prefect/pull/12798
+- Migrate `prefect-azure` docs to Integrations section of the Prefect docs — https://github.com/PrefectHQ/prefect/pull/12794
+- Update storage guide credentials blocks — https://github.com/PrefectHQ/prefect/pull/12819
+- Remove `server` import recommendations — https://github.com/PrefectHQ/prefect/pull/12823
+- Remove link to removed API page — https://github.com/PrefectHQ/prefect/pull/12824
+- Add Azure Container Instances worker guide — https://github.com/PrefectHQ/prefect/pull/12846
+- Improve wording on integrations index page — https://github.com/PrefectHQ/prefect/pull/12852
+
+### Integrations
+
+#### Prefect UI Library
+- Feature: `FormattedDate` component — https://github.com/PrefectHQ/prefect-ui-library/pull/2329
```

Comment:
Is there a way to make this sound more user-facing?

Reply:
I can come up with something.